packages/terra-form-fieldset/tests/nightwatch/fieldset-spec.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Do we need a nightwatch test? It's not doing much, and it looks like your wdio tests are doing what we need.

Reply:
+1 lets remove nightwatch tests when we can.